src/components/Dashboard/ProgressBar/ProgressBar.js

```diff
@@ -1,15 +1,45 @@
-import React from "react";
+import React, { Component } from "react";
 import styles from "./ProgressBar.scss";
 
-const ProgressBar = () => {
-  return (
-    <ul className={styles.ProgressBar}>
-      <li>Dane dziecka</li>
-      <li>Wnioskodawcy</li>
-      <li>Zespół</li>
-      <li>Podsumowanie</li>
-    </ul>
-  );
-};
+class ProgressBar extends Component {
+  render() {
+    let menuStep1;
+    let menuStep2;
+    let menuStep3;
+    let menuStep4;
+
+    // eslint-disable-next-line react/prop-types
+    switch (this.props.pageCounter) {
```

Comment:
We could be smart about that. Instead of big switch like that, we could just create a method for checking if we should apply `active` class to element. like `const isActive = (pageNumber) => this.props.pageCounter >= pageNumber`. Then call this method in JSX for `li` elements